docs/how-to-work-on-the-docs-theme.md

```diff
@@ -53,13 +53,7 @@ git clone https://github.com/freeCodeCamp/freeCodeCamp.git
 Install `docsify`:
 
 ```console
-npm install -g docsify
-```
-
-and serve the `/docs` directory
-
-```console
-docsify serve docs
+npx docsify serve docs
```

Comment:
I have changed this, because I think in Gitpod the `-g` flag is stopped.

So I changed it to npx, to run it